### PR TITLE
Feature: configure: make configure.set to update operation

### PR DESCRIPTION
**Pull request by liangxin1300**
#### Problem
Like issue #474 mentioned, it's not convenient for modifying an existing resource op monitor parameter.
Using `edit` can modify it but it's not work if user want to script this process

#### Solution
This PR extend the functionalities of configure.set subcommand, while try to keep compatible, if we want to update/add op parameters for en existing resource, we can do it like this:
```
set vip1.monitor.on-fail ignore
set drbd.monitor.10s.interval 20s
```
usage for configure.set
```
set <path> <value>

path:: id.[op_type.][interval.]name
```

#### Test
Behave test file is [here](https://github.com/ClusterLabs/crmsh/blob/38047ca78b46164b9f65b8d448cc66e15971b972/test/features/resource_set.feature)
include validation and normal using for configure.set

### CODE REVIEW

File: crmsh/ui_configure.py [hunk 1]
```diff
@@ -592,24 +592,47 @@ def do_filter(self, context, filterprog, *args):
...
-            context.fatal_error("Resource %s not found" % (obj_id))
-        nvpairs = rsc.node.xpath(".//nvpair[@name='%s']" % (obj_attr))
-        if not nvpairs:
-            context.fatal_error("Attribute not found: %s" % (path))
-        if len(nvpairs) != 1:
-            context.fatal_error("Expected 1 attribute named %s, found %s" %
-                                (obj_attr, len(nvpairs)))
+            context.fatal_error("Object {} not found".format(obj_id))
+
+        if len(other_path_list) == 1:
```
Comment (arbulu89) [inline crmsh/ui_configure.py]: Could we put some comment in top of the if to show which is the use case. Like `# id.name value use case`

File: crmsh/ui_configure.py [hunk 2]
```diff
@@ -592,24 +592,47 @@ def do_filter(self, context, filterprog, *args):
...
+            obj_attr = other_path_list[0]
+            nvpairs = rsc.node.xpath(".//nvpair[@name='{}']".format(obj_attr))
+            if not nvpairs:
+                context.fatal_error("Attribute not found: {}".format(path))
+            if len(nvpairs) != 1:
+                context.fatal_error("Expected 1 attribute named {}, found {}".format(obj_attr, len(nvpairs)))
+            nvpairs[0].set("value", value)
+
+        if len(other_path_list) == 2:
```
Comment (arbulu89) [inline crmsh/ui_configure.py]: The same about the comment. `# id.op_type.name value use case`

File: crmsh/ui_configure.py [hunk 3]
```diff
@@ -592,24 +592,47 @@ def do_filter(self, context, filterprog, *args):
...
+                context.fatal_error("Operation \"{}\" not found for resource {}".format(op_type, obj_id))
+            if len(op_res) > 1:
+                context.fatal_error("Should specify interval of {}".format(op_type))
+            op_res[0].set(name, value)
+
+        if len(other_path_list) == 3:
+            op_type, iv, name = other_path_list
+            op_res = rsc.node.xpath(".//operations/op[@id='{}-{}-{}']".format(obj_id, op_type, iv))
+            if not op_res:
+                context.fatal_error("Operation \"{}\" interval \"{}\" not found for resource {}".format(op_type, iv, obj_id))
```
Comment (arbulu89) [inline crmsh/ui_configure.py]: Don't we have here the case where the `len(op_res) > 1` like in the others?

File: crmsh/ui_configure.py (hunk 1 above)
Comment (liangxin1300) [reply crmsh/ui_configure.py]: Changed, thanks!

File: crmsh/ui_configure.py (hunk 2 above)
Comment (liangxin1300) [reply crmsh/ui_configure.py]: Changed, thanks!

File: crmsh/ui_configure.py (hunk 3 above)
Comment (liangxin1300) [reply crmsh/ui_configure.py]: `len(op_res) > 1` means these Ids are exactly same, that is impossible for cib, it's invalid

**Comment by liangxin1300** [comment]
Thanks @arbulu89! 
I've also updated backport PR #515 
Please approve both of them

**Pull request merged**